lib/generate.js

```diff
@@ -183,6 +183,7 @@ function generateScripts () {
 
 async function generateAll () {
     await ensureDir(CONFIG.publicDir);
+    ensureSymlink('.', `./${CONFIG.publicDir}/helix-ui`);
```

Comment:
`localhost:300` and `localhost:3000/helix-ui` and `localhost:3000/helix-ui/helix-ui`, etc. all resolve to the same physical directory